lib/pbench/test/unit/server/conftest.py

```diff
@@ -394,6 +394,32 @@ def provide_metadata(attach_dataset):
     )
     Metadata.setvalue(dataset=test, key="dashboard.contact", value="you@example.com")
     Metadata.setvalue(dataset=test, key=Metadata.DELETION, value="2023-01-25")
+    Metadata.create(
+        dataset=drb,
+        key=Metadata.METALOG,
+        value={
+            "pbench": {
+                "date": "2020-02-15T00:00:00",
+                "config": "test1",
+                "script": "unit-test",
+                "name": "drb",
+            },
+            "run": {"controller": "node1.example.com"},
+        },
+    )
+    Metadata.create(
+        dataset=test,
```

Comment:
The various steps of the instantiations of the `drb` and `test` datasets are kind of arbitrarily intertwined -- it would be nicer if they were either separated or performed in alternating sequence (e.g., by key).

Reply:
Basically these new ones were unavoidably multi-line and at the time I decided it was "neater" looking to segregate those at the end. However, yeah; I put in an `index-map` for `drb` (hey, my sample datasets are ahead of their time: freeform "metadata" name instead of canonical `<script>_<config>_<date>`; [PBENCH-744](https://issues.redhat.com/browse/PBENCH-744)). I'll move the `drb` metalog up with it.